apps/demo-ui/run-ui.sh

```diff
@@ -27,20 +27,33 @@ function cleanup {
 
 cd "${PROXYDIR}"
 : ${HOST:=localhost}
-if [[ (! -f ${HOST}-key.pem) || (! -f ${HOST}-cert.pem) ]]; then
+SELFSIGNED="${HOST}_ss"
+SSLNAME="${SELFSIGNED}"
+ARYN_DIR=/etc/opt/aryn
```

Comment:
I'd call this ARYN_ETC 
(I would have thought of /opt/aryn as ARYN_DIR)

Reply:
Sure.